tools/manta/manta.xml

```diff
@@ -1,292 +1,293 @@
+		ln -s '$bam_input.normal_bam_file' $input_normal &&
+		ln -s '$bam_input.normal_bam_file.metadata.bam_index' normal.bai &&
+		ln -s '$bam_input.tumor_bam_file' $input_tumor &&
+		ln -s '$bam_input.tumor_bam_file.metadata.bam_index' tumor.bai &&
+		#end if
+
+		cp ${config_file} ${config_file_custom} &&
+
+		#if str( $set_configuration.set_configuration_switch ) == "Customized":
+		sed -i 's/minCandidateVariantSize = 8/minCandidateVariantSize = $set_configuration.minCandidateVariantSize/' ${config_file_custom} &&
+		sed -i 's/rnaMinCandidateVariantSize = 1000/rnaMinCandidateVariantSize = $set_configuration.rnaMinCandidateVariantSize/' ${config_file_custom} &&
+		sed -i 's/minEdgeObservations = 3/minEdgeObservations = $set_configuration.minEdgeObservations/' ${config_file_custom} &&
+		sed -i 's/graphNodeMaxEdgeCount = 10/graphNodeMaxEdgeCount = $set_configuration.graphNodeMaxEdgeCount/' ${config_file_custom} &&
+		sed -i 's/minCandidateSpanningCount = 3/minCandidateSpanningCount = $set_configuration.minCandidateSpanningCount/' ${config_file_custom} &&
+		sed -i 's/minScoredVariantSize = 50/minScoredVariantSize = $set_configuration.minScoredVariantSize/' ${config_file_custom} &&
+		sed -i 's/minDiploidVariantScore = 10/minDiploidVariantScore = $set_configuration.minDiploidVariantScore/' ${config_file_custom} &&
+		sed -i 's/minPassDiploidVariantScore = 20/minPassDiploidVariantScore = $set_configuration.minPassDiploidVariantScore/' ${config_file_custom} &&
+		sed -i 's/minPassDiploidGTScore = 15/minPassDiploidGTScore = $set_configuration.minPassDiploidGTScore/' ${config_file_custom} &&
+		sed -i 's/minSomaticScore = 10/minSomaticScore = $set_configuration.minSomaticScore/' ${config_file_custom} &&
+		sed -i 's/minPassSomaticScore = 30/minPassSomaticScore = $set_configuration.minPassSomaticScore/' ${config_file_custom} &&
+		sed -i 's/enableRemoteReadRetrievalForInsertionsInGermlineCallingModes = 1/enableRemoteReadRetrievalForInsertionsInGermlineCallingModes = $set_configuration.enableRemoteReadRetrievalForInsertionsInGermlineCallingModes/' ${config_file_custom} &&
+		sed -i 's/enableRemoteReadRetrievalForInsertionsInCancerCallingModes = 0/enableRemoteReadRetrievalForInsertionsInCancerCallingModes = $set_configuration.enableRemoteReadRetrievalForInsertionsInCancerCallingModes/' ${config_file_custom} &&
+		sed -i 's/useOverlapPairEvidence = 0/useOverlapPairEvidence = $set_configuration.useOverlapPairEvidence/' ${config_file_custom} &&
+		#end if
+
+		configManta.py
+		--referenceFasta='${reference_fasta_filename}'
+
+		#if str( $set_configuration.set_configuration_switch ) == "Custom_config_file":
+		#set config_file = $set_configuration.CustomConfigFile
+		#else if str( $set_configuration.set_configuration_switch ) == "Customized":
+		#set config_file = config_file_custom
+		#end if
+
+		--config=${config_file}
+
+		#if str( $bam_input.bam_input_selector ) == "not_tumor_bam":
+		--bam=$input_normal
```

Comment:
The indentation is not correct here, 4 spaces to the right following the python convention

Reply:
Ok, I will correct it.